docs/intro/docker.md

```diff
@@ -0,0 +1,79 @@
+
+## Prerequisites
+
+Docker must be installed. See [Get Started with Docker](https://www.docker.com/get-started).
+
+## Scaffolding a Chain
+
+When Docker is installed, you can build a blockchain with a single command.
+
+To scaffold a blockchain `planet` in a new directory, run this command in a terminal window:
+
+```
+docker run -ti -v $HOME/sdh:/home/tendermint -v $PWD:/apps starport/cli:0.16.0 app github.com/hello/planet
+```
+
+Be patient, this command takes a minute or two to run because it does everything for you:
+
+- Creates a container that runs from the `starport/cli:0.16.0` image.
+- Executes the Starport binary inside the image.
+- Uses `-v $HOME/sdh:/home/tendermint` to map the `$HOME/sdh` directory in your local computer (the host machine) to the home directory `/home/tendermint` inside the container. Starport, and the chains you serve with Starport, persist some files.
+- Uses `-v $PWD:/apps` to persist the scaffolded app in the container to the host machine at current working directory.
+
+## Starting a Blockchain
+
+To start the blockchain node in the Docker container you just created, run this command:
+
+```
+docker run -ti -v $HOME/sdh:/home/tendermint -v $PWD:/apps -p 1317:1317 -p 26657:26657 starport/cli:develop serve -p planet
+```
+
+This command does the following:
+
+- `-v $HOME/sdh:/home/tendermint` maps the `$HOME/sdh` directory in your local computer (the host machine) to the home directory `/home/tendermint` inside the container.
+- `-v $PWD:/apps` persists the scaffolded app in the container to the host machine at current working directory.
+- `serve -p planet` specifies to use the `planet` directory that contains the source code of the blockchain.
+- `-p 26657:26657` maps RPC server port 26657 (tendermint) on the host machine to port 26657 in Docker. This mapping exposes ports from the container to the host machine.
+- `-p 1317:1317` maps the API server port (cosmos-sdk) to the host machine
+- After the blockchain is started, open `http://localhost:26657` to see the Tendermint API.
+- The `-v` flag specifies for the container to access the application's source code from the host machine so it can build and run it.
+
+## Versioning
+
+You can specify which version of Starport to install and run in your Docker container.
+
+### Latest Version
+
+- By default, `starport/cli` resolves to `starport/cli:latest`.
+- The `latest` image tag is always the latest stable [Starport release](https://github.com/tendermint/starport/releases).
+
+For example, if latest release is [v0.15.1](https://github.com/tendermint/starport/releases/tag/v0.15.1), the `latest` tag points to the `0.15.1` tag.
+
+### Specific Version
+
+You can set the version tag to use a specific version. All available tags are in the [starport/cli image](https://hub.docker.com/repository/docker/starport/cli/tags?page=1&ordering=last_updated) on Docker Hub.
+
```

Comment:
looks like all of the tags other than `latest` were deleted

Reply:
Once we release v0.16 it'll be pushed to Docker Hub.